tests/opt-testcases/tls13-kex-modes.sh

```diff
@@ -66,6 +66,233 @@ run_test    "TLS 1.3: PSK: No valid ciphersuite. O->m" \
             -s "Found PSK KEX MODE" \
             -s "No matched ciphersuite"
 
+requires_all_configs_enabled MBEDTLS_SSL_PROTO_TLS1_3 MBEDTLS_SSL_SESSION_TICKETS MBEDTLS_SSL_SRV_C \
+                             MBEDTLS_SSL_CLI_C MBEDTLS_DEBUG_C MBEDTLS_HAVE_TIME
+requires_any_configs_enabled MBEDTLS_KEY_EXCHANGE_PSK_ENABLED MBEDTLS_KEY_EXCHANGE_RSA_PSK_ENABLED \
+                             MBEDTLS_KEY_EXCHANGE_DHE_PSK_ENABLED MBEDTLS_KEY_EXCHANGE_ECDHE_PSK_ENABLED
+requires_any_configs_enabled MBEDTLS_KEY_EXCHANGE_PSK_ENABLED MBEDTLS_KEY_EXCHANGE_RSA_PSK_ENABLED \
+                             MBEDTLS_KEY_EXCHANGE_DHE_PSK_ENABLED MBEDTLS_KEY_EXCHANGE_ECDHE_PSK_ENABLED
+run_test "TLS 1.3: NewSessionTicket: psk_ephemeral, auth fail, m->m" \
```

Comment:
I would decouple the tests about one PSK plus one ticket from the tests over the various values of `dummy_ticket`. Thus for one PSK plus one ticket I would say one test where the ticket is selected and one test where the PSK is selected because the ticket is not valid, both with `tls13_kex_modes=psk_ephemeral`.

Then for the tests over the possible values of `dummy_ticket` I would suggest only one ticket, no external PSK, with `tls13_kex_modes=psk` and the handshake failing.

Reply:
> Then for the tests over the possible values of `dummy_ticket` I would suggest only one ticket, no external PSK, with `tls13_kex_modes=psk` and the handshake failing.

A difference, `tls13_kex_modes=psk_or_ephemeral` . We should get ticket with `ephemeral` mode and then reconnect with psk mode.

Reply:
> without `ephemeral` or PSK key, we can not get ticket.

Good point.